lib/AppInfo/Application.php

```diff
@@ -59,6 +60,7 @@
 use OCA\Mail\Service\UserPreferenceSevice;
 use OCP\AppFramework\App;
 use OCP\AppFramework\IAppContainer;
+use OCP\Dashboard\RegisterWidgetEvent;
```

Comment:
I think we need 19+ for this, but currently we allow https://github.com/nextcloud/mail/blob/424624f24d864b372275f2938c6ab7bb17fd673c/appinfo/info.xml#L34

Reply:
it will work with older releases as well as we only use it as string :+1: